src/media/js/addon/components/addon.js

```diff
@@ -60,15 +60,39 @@ export class Addon extends React.Component {
           {this.renderName()}
         </div>
         <dl>
-          <dt>Slug</dt>
-          <dd>{this.props.slug}</dd>
+          <di>
+            <dt>Slug</dt>
+            <dd>{this.props.slug}</dd>
+          </di>
+
+          <di>
+            <dt>Manifest</dt>
```

Comment:
It doesn't really matter right now, it's good enough to meet the story this week, adn we'll have designs soon enough.